browser/src/control/Control.MobileTopBar.js

```diff
@@ -122,18 +122,20 @@ L.Control.MobileTopBar = L.Control.extend({
 
 	onUpdatePermission: function(e) {
 		var toolbarButtons = ['next', 'prev', 'mobile_wizard', 'insertion_mobile_wizard', 'comment_wizard'];
+		// Handle permission mode as diffrent case because it's not part of `toolbar-up` element
+		var PermissionMode = document.getElementById('PermissionMode');
 		if (e.perm === 'edit') {
 			toolbarButtons.forEach((id) => {
 				this.enableItem(id, true);
 			});
-			this.showItem('PermissionMode', false);
+			PermissionMode.style.display = 'none';
```

Comment:
`showItem` didn't work as expected? I prefer to use high level abstration. (if it doesn't work it's ok, it's HTML content we need to convert in future into some "native" widget)

Reply:
The problem is `PermissionMode` is not part of parent Container, so that is why i change showItem to simple html hide

Reply:
We can change here showItem method a bit to pass parent container and then can use to show or hide element using ShowItem. @eszkadev thoughts ?

Reply:
Something like this
![image](https://github.com/CollaboraOnline/online/assets/61383886/cba87d0b-a687-4318-9609-5571aa7bd48a)

here for element like permissionMode which now extracted outside (#Toolbar-up) can have it's own parent to pass on to showItem